homeassistant/components/squeezebox/media_player.py

```diff
@@ -85,13 +95,28 @@
     }
 )
 
+SQUEEZEBOX_SYNC_SCHEMA = MEDIA_PLAYER_SCHEMA.extend(
+    {vol.Required(ATTR_OTHER_PLAYER): cv.string}
+)
+
 SERVICE_TO_METHOD = {
     SERVICE_CALL_METHOD: {
         "method": "async_call_method",
         "schema": SQUEEZEBOX_CALL_METHOD_SCHEMA,
-    }
+    },
+    SERVICE_CALL_QUERY: {
+        "method": "async_call_query",
+        "schema": SQUEEZEBOX_CALL_METHOD_SCHEMA,
+    },
+    SERVICE_SYNC: {"method": "async_sync", "schema": SQUEEZEBOX_SYNC_SCHEMA},
+    SERVICE_UNSYNC: {"method": "async_unsync", "schema": None},
```

Comment:
We should use our entity service helper instead.

https://developers.home-assistant.io/docs/dev_101_services#entity-services